VideoRenderer/VideoRenderer/SystemPlayerObserver.swift

```diff
@@ -119,9 +135,15 @@ public final class SystemPlayerObserver: NSObject {
                         self,
                         name: .AVPlayerItemDidPlayToEndTime,
                         object: old)
+                    if let token = accessLogToken {
```

Comment:
We can have a bug here if we will replace not nil `AVPlayer.currentItem` with new.  In such case `.new`value comes firs, `accessLogToken` will be overwritten, and when we get there with `oldItem`, there will be no match between new `accessLogToken` and old `oldItem`, notification will not be removed.